ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -84,15 +84,69 @@ releases:
       stream: nightly
       version: "4.16"
   oldest-supported-z-rollback:
-    release:
-      channel: candidate
-      version: 4.16.0-ec.0
+    candidate:
+      product: ocp
+      relative: 1
```

Comment:
shall we use latest ec build?

Reply:
ec build is not that reliable, please see my comment https://github.com/openshift/release/pull/48427#issuecomment-2003344080
I'll change it to rc.0 once it's ready.

Reply:
i'm thinking if two nightly builds with only one hop are too close. and if rc.0 is not reliable, then that means there should be next rc.

Reply:
I meant EC is not reliable. I'll use RC.0 instead later. Rolling back between 2 nightlies are fine.